CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/CompleteGraphBuilderPlugin.java

```diff
@@ -272,22 +270,18 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
             }
         }
 
-        if (!PreferenceUtilites.isGraphViewFrozen()) {
-            try {
-                if (n < 10000) {
-                    // Do a trees layout.
-                    PluginExecutor.startWith(ArrangementPluginRegistry.TREES)
-                            .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
-                } else {
-                    // Do a grid layout.
-                    PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
-                            .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
-                }
-            } catch (PluginException ex) {
-                Exceptions.printStackTrace(ex);
+        try {
+            if (n < 10000) {
+                // Do a trees layout.
+                PluginExecutor.startWith(ArrangementPluginRegistry.TREES)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+            } else {
+                // Do a grid layout.
+                PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
+                        .followedBy(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
             }
-        } else {
-            PluginExecution.withPlugin(InteractiveGraphPluginRegistry.RESET_VIEW).executeNow(graph);
+        } catch (PluginException ex) {
```

Comment:
I know this was just a re-format but would be good if you could make this final

Reply:
Added a final